doc/toolbox.1.md

```diff
@@ -48,6 +48,9 @@ falls back to a Fedora image. Supported host operating systems are:
 
 * Fedora
 * Red Hat Enterprise Linux >= 8.5
+* CentOS Stream
+* Arch Linux
```

Comment:
We don't support CentOS Stream, because we don't have built-in images wired with `--distro` for it, but we do support Ubuntu.  There's a table below that needs updating too.

Also, please keep the lists alphabetically ordered.  :)